src/assets/styles/_variables.scss

```diff
@@ -0,0 +1,62 @@
+// --------------------------
+// COLORS
+// --------------------------
+
+$cream: hsl(50, 81%, 77%);
```

Comment:
Thanks so much for setting these up they'll be incredibly helpful while I style today!